docs/conf.py

```diff
@@ -348,9 +347,8 @@
 #    }
 
 rst_epilog = '''
-.. |openstack| replace:: %(openstack_release)s
 .. |f5_agent_pip_url| replace:: git+https://github.com/F5Networks/f5-openstack-agent@v%(version)s
-.. |f5_agent_pip_url_branch| replace:: git+https://github.com/F5Networks/f5-openstack-agent@%(openstack_release_l)s
+.. |f5_agent_pip_url_branch| replace:: git+https://github.com/F5Networks/f5-openstack-agent
```

Comment:
I forgot this one -- we should take out this line and the section in the install doc that references it: